docs/api.md

```diff
@@ -552,6 +552,79 @@ Output content of the table to a `pandas.DataFrame`.
     
     Maximum number of rows to return. Defaults to -1, i.e. all.
 
+## API Client
+
+
+### `sematic.client.block_on_run`
+
+
+Block on the run with the given id until it is in a terminal state.
+
+Terminal states include successful completion, failure and cancellation.
+Only successful completion returns without error, other terminal states
+result in `RuntimeError` being raised.
+
+#### Parameters
+
+- `run_id`: str
+
+    The id of the run to block on.
+
+- `delay_seconds`: float
+
+    The number of seconds between polling for updates to the run's status.
+
+- `max_wait_seconds`: float
+
+    If the run has not terminated after this number of seconds, will raise
+    `TimeoutError`. If this is `None`, will wait indefinitely. Note that
+    if `block_on_run` has failed with a timeout, this does NOT mean the run
+    itself has failed or timed out; the run may continue unimpacted. This
+    timeout is only for the block call itself.
+
+#### Raises
+
+`RuntimeError`
+
+If the run does not complete successfully.
```

Comment:
To definitely disambiguate from timeout:
```suggestion
If the run completes with a failure.
```

Reply:
A cancellation is not a failure, but will still result in a `raise`